docs/ksql-clickstream-demo/non-docker-clickstream.rst

```diff
@@ -0,0 +1,364 @@
+        Query terminated    
+
+14. Go to your terminal and send the KSQL tables to Elasticsearch and
+    Grafana.
+
+    1. From your terminal, navigate to the demo directory:
+
+       ::
+
+           cd ksql-clickstream-demo/demo/
+
+    2. Run this command to send the KSQL tables to Elasticsearch and
+       Grafana:
+
+       .. code:: bash
+
+           $ ./ksql-tables-to-grafana.sh
+
+       Your output should resemble:
+
+       .. code:: bash
+
+           Loading Clickstream-Demo TABLES to Confluent-Connect => Elastic => Grafana datasource
+           Logging to: /tmp/ksql-connect.log
+           Charting  CLICK_USER_SESSIONS_TS
+           Charting  USER_IP_ACTIVITY_TS
+           Charting  CLICKSTREAM_STATUS_CODES_TS
+           Charting  ENRICHED_ERROR_CODES_TS
+           Charting  ERRORS_PER_MIN_ALERT_TS
+           Charting  ERRORS_PER_MIN_TS
+           Charting  EVENTS_PER_MIN_MAX_AVG_TS
+           Charting  EVENTS_PER_MIN_TS
+           Charting  PAGES_PER_MIN_TS
+           Navigate to http://localhost:3000/dashboard/db/click-stream-analysis
+
+       **Important:** The ``http://localhost:3000/`` URL is only
+       available inside the container. We will access the dashboard with
+       a slightly different URL, after running the next command.
+
+    3. From your terminal, load the dashboard into Grafana.
+
+       .. code:: bash
+
+           $ ./clickstream-analysis-dashboard.sh
+
+       Your output should resemble:
+
+       .. code:: bash
+
+           Loading Grafana ClickStream Dashboard
+           {"slug":"click-stream-analysis","status":"success","version":1}
+
+15. Go to your browser and view the Grafana output at
+    http://localhost:3000/dashboard/db/click-stream-analysis. You can
+    login with user ID ``admin`` and password ``admin``.
+
+    **Important:** If you already have Grafana UI open, you may need to
+    enter the specific clickstream URL:
+    http://localhost:3000/dashboard/db/click-stream-analysis.
+
+    .. figure:: grafana-success.png
+       :alt: Grafana UI success
+
+       Grafana UI success
+
+| Interesting things to try: \* Understand how the
+  ``clickstream-schema.sql`` file is structured. We use a
+  DataGen.KafkaTopic.clickstream -> Stream -> Table (for window &
+  analytics with group-by) -> Table (to Add EVENT_TS for time-index) ->
+  ElasticSearch/Connect topic
+| \* Run the ``LIST TOPICS;`` command to see where data is persisted \*
+  Run the KSQL CLI ``history`` command
+
+Troubleshooting
+---------------
+
+-  Docker must not be running on the host machine.
+-  Check that Elasticsearch is running: http://localhost:9200/.
+-  Check the Data Sources page in Grafana.
+
+   -  If your data source is shown, select it and scroll to the bottom
+      and click the **Save & Test** button. This will indicate whether
+      your data source is valid.
+   -  If your data source is not shown, go to
+      ``/ksql/ksql-clickstream-demo/demo/`` and run
```

Comment:
I think the absolute path `/ksql/ksql-clickstream-demo/demo/` is incorrect.  Should this be `<path-to-ksql>/ksql-clickstream-demo/demo/`?